opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicMetricExporter.java

```diff
@@ -113,8 +113,9 @@ public CompletableResultCode flush() {
   }
 
   @Override
-  public void shutdown() {
+  public CompletableResultCode shutdown() {
     telemetryClient.shutdown();
+    return CompletableResultCode.ofSuccess();
```

Comment:
What's the purpose of returning this if the result is always success?

Reply:
I find this to be a bit confusing as well. We use the same pattern in the `NewRelicSpanExporter` and for other methods that are overridden from the `SpanExporter` interface.

We don't actually have access to any kind of result from here as we delegate to `telemetryClient.shutdown()` and it doesn't return anything that indicates success/failure. The `TelemetryClient` also swallows up any exceptions, so I guess from the perspective of our `NewRelicSpanExporter` and `NewRelicMetricExporter` it's always a successful shutdown. So even though returning `CompletableResultCode.ofSuccess()` doesn't seem to add any real value it seems correct and meets the requirements of the interface that it is implementing.

I'm not opposed to maybe rethinking/refactoring this in a different PR but I'd rather not muddy up the purpose of this PR by going down that route.

Reply:
Oh gotcha didn't see the `@Override` annotation.